pyccel/parser/semantic.py

```diff
@@ -2710,6 +2711,52 @@ def _visit_ListExtend(self, expr):
             for_obj = For(for_target, iterable, body)
             pyccel_stage.set_stage('semantic')
             return self._visit(for_obj)
+    
+    def _visit_SetUpdate(self, expr):
```

Comment:
Please can you move this function to the bottom of the file. This will help with the diff when I merge it into #1844

Reply:
Thanks :)